src/Symfony/Component/EventDispatcher/Tests/CompiledEventDispatcherTest.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\EventDispatcher\Tests;
+
+use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\EventDispatcher\CompiledEventDispatcher;
+
+class CompiledEventDispatcherTest extends AbstractEventDispatcherTest
+{
+    protected function createEventDispatcher()
+    {
+        $container = new Container();
+
+        return new CompiledEventDispatcher($container);
+    }
+
+    public function testGetListenersWithCallables()
+    {
+        // When passing in callables exclusively as listeners into the event
+        // dispatcher constructor, the event dispatcher must not attempt to
+        // resolve any services.
+        $container = $this->getMock('Symfony\Component\DependencyInjection\IntrospectableContainerInterface');
+        $container->expects($this->never())->method($this->anything());
```

Comment:
`createEventDispatcher` overrides the abstract method of `AbstractEventDispatcherTest`.